nexus/reconfigurator/planning/src/example.rs

```diff
@@ -190,7 +190,7 @@ impl ExampleSystemBuilder {
     pub fn build(&self) -> (ExampleSystem, Blueprint) {
         let nexus_count = self.get_nexus_zones();
 
-        slog::info!(
+        slog::trace!(
```

Comment:
I'm guessing you changed this so that `reconfigurator-cli` doesn't print this out?

But I think this is could be invaluable in _post hoc_ debugging test failures and I'm worried we might not always have it.  Can we leave it at `debug!`?

Reply:
Yeah -- I tried `debug!` but reconfigurator-cli logs debug messages to stderr. What do you think of changing the default log level of reconfigurator-cli to info?

Reply:
I think that's fine.  If by construction we don't want it to see these messages... maybe it should even be `warn`?

Reply:
Let's do info for now, I'm not hugely confident warn is right

Reply:
Changed this to debug and reconfigurator-cli's filter to info.